src/ol_openedx_course_structure_api/views.py

```diff
@@ -86,6 +83,21 @@ def get(self, request, course_id):
         if inherited_metadata:
             compute_inherited_metadata(course)
 
+        # dump_block is renamed from dump_module in release-2023-03-08-14.35
+        # For backward compatibility, try import the new name and then old name
+        try:
```

Comment:
Should this be done in the top level imports, rather than in the context of the function execution?

Reply:
I did it inside right before `dump_block` is called. Probably can be done in top level as well. I can test. Is it preferred to be done in top level?

Reply:
Moved to the top level and tested